client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -191,8 +191,20 @@ public class RssSparkConfig {
       createIntegerBuilder(new ConfigBuilder("spark.rss.client.unregister.thread.pool.size"))
           .createWithDefault(10);
 
+  public static final ConfigEntry<Integer> RSS_CLIENT_UNREGISTER_TIMEOUT_SEC =
+      createIntegerBuilder(
+              new ConfigBuilder("spark.rss.client.unregister.timeout.sec")
+                  .doc(
+                      "Unregister requests are executed concurrently and all requests together "
+                          + "have to complete within this timeout."))
+          .createWithDefault(60);
```

Comment:
This default effectively changes current behaviour waiting for all requests from 10s to 60s. Should we leave the default at 10s? Is having a 1 minute graceful shutdown reasonable for everyone? There will be a warning if this timeout occurs:

    Timeout occurred while unregistering from {}. Please consider increasing the thread pool size (10) or the overall timeout (10s) if you still think the request timeout (10s) is sensible.

Reply:
Please use the `ConfigOption` to init this config. The `configEntry` is not a recommended way

Reply:
I think we'd better to reserve the original 10s timeout, if this is too short, let's optimize after having some user cases. WDYT?

Reply:
> Please use the ConfigOption to init this config. The configEntry is not a recommended way

I know, but all options in this file use `ConfigEntry`, refactoring this is part of #1302, sticking to the current coding style rather than mixing coding styles here.

Reply:
Yes. But I hope the newly added config options could be obey the newer rule, that will reduce the later refactor pressure.